src/footer/footer.tsx

```diff
@@ -129,15 +130,21 @@ export const Footer = <T,>({
 
         if (component) {
             return (
-                <TopSection type="grid" stretch={isStretch}>
-                    {component}
-                </TopSection>
+                <>
+                    <TopSection type="grid" stretch={isStretch}>
+                        {component}
+                    </TopSection>
+                    <Divider
+                        style={{
+                            width: "100%",
+                        }}
+                    />
```

Comment:
the divider should be placed in the TopSection which has the grid layout

and it requires top margin, which should be applied in a styled component

```
<StyledDivider layoutType="grid" xxsCols={[1, -1]} />
```

Reply:
to also remove `MobileOnlyBorder`